### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by sofianehaddad**
Issue in documentation that leads user to bad usage & understanding 

```
Backword --> 1
Forward --> 2
Both --> 3
```

### CODE REVIEW

**Comment by jschueller** [comment]
careful you're pushing to the main repo

**Comment by jschueller** [comment]
included in #1941

File: python/src/LinearModelStepwiseAlgorithm_doc.i.in [hunk 1]
```diff
@@ -64,7 +64,7 @@ Creation of a basis
 Stepwise regression
 
 >>> minimalIndices = [0]
->>> direction = 1 #Forward selection
+>>> direction = 1 #Backward selection
```
[the remark this answers is not in the document]
Comment (sofianehaddad) [reply python/src/LinearModelStepwiseAlgorithm_doc.i.in:67]: we could indeed

Comment (sofianehaddad) [inline python/src/LinearModelStepwiseAlgorithm_doc.i.in:67]:
```suggestion
>>> direction = LinearModelStepwiseAlgorithm.BACKWARD
```

**Pull request closed without merging**